diff -crB /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/etc/board.d/02_network ../openwrt/target/linux/ramips/base-files/etc/board.d/02_network
*** /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/etc/board.d/02_network	2017-07-09 02:18:55.513986080 +0800
--- ../openwrt/target/linux/ramips/base-files/etc/board.d/02_network	2017-07-09 02:35:14.787942419 +0800
***************
*** 185,190 ****
--- 185,196 ----
  		ucidef_add_switch_vlan "switch0" "2" "0 6t"
  		;;
  
+ a6)
+ 		ucidef_set_interface_lan "eth0.1"
+ 		ucidef_add_switch "switch0" "1" "1"
+ 		ucidef_add_switch_vlan "switch0" "1" "0 1 2 3 4 6t"
+ 		;;
+ 
  	ur-336un)
  		ucidef_set_interfaces_lan_wan "eth0.1" "eth0.2"
  		;;
diff -crB /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/lib/ramips.sh ../openwrt/target/linux/ramips/base-files/lib/ramips.sh
*** /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/lib/ramips.sh	2017-07-09 02:18:55.513986080 +0800
--- ../openwrt/target/linux/ramips/base-files/lib/ramips.sh	2017-07-09 02:32:53.314948727 +0800
***************
*** 355,360 ****
--- 355,363 ----
  	*"Tenda W150M")
  		name="w150m"
  		;;
+     *"Tenda A6")
+         name="a6"
+         ;;
  	*"TEW-691GR")
  		name="tew-691gr"
  		;;
diff -crB /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/lib/upgrade/platform.sh ../openwrt/target/linux/ramips/base-files/lib/upgrade/platform.sh
*** /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/base-files/lib/upgrade/platform.sh	2017-07-09 02:18:55.513986080 +0800
--- ../openwrt/target/linux/ramips/base-files/lib/upgrade/platform.sh	2017-07-09 02:37:05.690937475 +0800
***************
*** 20,25 ****
--- 20,26 ----
  	3g300m | \
  	a5-v11 | \
  	air3gii | \
+     a6 | \
  	ai-br100 |\
  	all0239-3g | \
  	all0256n | \
Only in ../openwrt/target/linux/ramips/dts: A6.dts
diff -crB /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/image/Makefile ../openwrt/target/linux/ramips/image/Makefile
*** /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/image/Makefile	2017-07-09 02:18:55.522986079 +0800
--- ../openwrt/target/linux/ramips/image/Makefile	2017-07-09 02:38:28.900933765 +0800
***************
*** 691,698 ****
--- 691,701 ----
  
  Image/Build/Profile/XDXRN502J=$(call BuildFirmware/Default4M/$(1),$(1),xdxrn502j,XDXRN502J)
  
+ Image/Build/Profile/A6=$(call BuildFirmware/Default4M/$(1),$(1),a6,A6)
+ 
  ifeq ($(SUBTARGET),rt305x)
  define Image/Build/Profile/Default
+         $(call Image/Build/Profile/A6,$(1))
  	$(call Image/Build/Profile/3G6200N,$(1))
  	$(call Image/Build/Profile/3G6200NL,$(1))
  	$(call Image/Build/Profile/3G150B,$(1))
diff -crB /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/rt305x/profiles/tenda.mk ../openwrt/target/linux/ramips/rt305x/profiles/tenda.mk
*** /mnt/pub3/src/openwrt/cc/openwrt/target/linux/ramips/rt305x/profiles/tenda.mk	2017-07-09 02:18:55.533986079 +0800
--- ../openwrt/target/linux/ramips/rt305x/profiles/tenda.mk	2017-07-09 02:39:34.115930857 +0800
***************
*** 41,43 ****
--- 41,55 ----
  endef
  
  $(eval $(call Profile,W306R_V20))
+ 
+ define Profile/A6
+ 	NAME:=Tenda A6
+ 	PACKAGES:=kmod-usb-core kmod-usb-ohci kmod-usb2 kmod-ledtrig-usbdev \
+ 		kmod-i2c-core kmod-i2c-gpio
+ endef
+ 
+ define Profile/A6/Description
+ 	Package set for Tenda A6
+ endef
+ 
+ $(eval $(call Profile,A6))
